javascript/javascript1/week4/hw4.js

```diff
@@ -0,0 +1,86 @@
+let todo = [];
+let myName = '';
+const month = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+function getReply(command) {
+
+    if (command.includes('Hello my name is Hamed Hello my name is Hamed')) {
+        return 'Please tell me just once your information';
+
+    } else if (command === 'Hello my name is Hamed') {
+        myName = command.split(' ');
+        return 'nice to meet you ' + myName[4];
+
+    } else if (command === 'Hello my name is') {
+        return 'Please tell your name'
+
+    } else if (command === 'What is my name?') {
+        return myName[4];
+
+    } else if (command === 'add fishing to my todo') {
+
+        todo.push(command.substring(4, 11));
+        return {
+            todo,
+            'firstTodo': 'fishing added to your todo'
+        };
+    } else if (command === 'add singing in the shower to my todo') {
+        todo.push(command.substring(4, 11))
+        return {
+            todo,
+            'secondTodo': 'singing added to your todo'
+        }
```

Comment:
How about finding all the words between "add" and "to my to do" and push them in `todo`? You can use `string.split(" ")`, `array.slice()` and `array.join(" ")` for that.

Reply:
Yes this is another way. Would you please let me know my way is incorrect or I am thinking in a worng way or I should change my solutions?And if my way is not correct let me know why?